crates/ln-dlc-node/src/node/ln_channel.rs

```diff
@@ -77,6 +79,18 @@ impl<P> Node<P> {
                         )
                         .map_err(|e| anyhow!("Could not force close channel {e:?}"))
                 } else {
+                    if let Some(SubChannel { state, .. }) = self.list_dlc_channels()?.first() {
+                        if *state != SubChannelState::OffChainClosed {
+                            // this is a safeguard to not close a ln channel with
+                            // an open dlc, as we would loose the funds otherwise.
```

Comment:
nit: typo ("loose" is a valid word, but means something different)

```suggestion
                            // an open dlc, as we would lose the funds otherwise.
```